cypress/e2e/pages/homePage/urls.js

```diff
@@ -11,6 +11,11 @@ const urls = [
     test: '/arabic',
     live: '/arabic',
   },
+  {
+    service: 'serbian',
+    local: '/serbian/lat',
+    test: '/serbian/lat',
```

Comment:
If there are any local/test E2E tests for variant services in cypress/support/config/settings.js, they can now be disabled e.g. https://github.com/bbc/simorgh/blob/latest/cypress/support/config/settings.js#L5663-L5672 - set disabled = false for both test & live environments, and repeat for ukchina & zhongwen

Reply:
Do we need to add in ukchina and zhongwen? Or is it a case of changing them if they already exist?

Reply:
I think the serbian homepage is the only one with decent content at the moment, but in general / rule of thumb, we don't want to run E2E tests against every single service because they are expensive / takes ages to run. We tend to select a handful of services which cover all bases e.g. LTR, RTL and service with variant. With kyrgyz, arabic & serbian, we should be sorted.

Reply:
Correct, just serbian - in fact, could we try to add both variants as separate entries in the array?